lib/st25r3911b/st25r3911b_common.c

```diff
@@ -526,24 +551,62 @@ int st25r3911b_field_on(u8_t collision_threshold, u8_t peer_threshold,
 	return -EACCES;
 }
 
-int st25r3911b_technology_led_set(u32_t led, bool on)
+int st25r3911b_technology_led_set(enum st25r3911b_leds led, bool on)
 {
 	int err;
+	gpio_pin_t pin;
+
+	switch (led) {
+	case ST25R3911B_NFCA_LED:
+		if (DT_NODE_HAS_PROP(ST25R3911B_NODE, led_nfca_gpios)) {
```

Comment:
You could use:
```
#if DT_NODE_HAS_PROP(ST25R3911B_NODE, led_nfca_gpios)
	case ST25R3911B_NFCA_LED:
```
to avoid those artificial definitions of:
```
#define NFCA_LED_PORT 0
#define NFCA_LED_PIN 0
```
and return `-ENOTSUP` only for `default:`.

Reply:
Done